src/Installer/index.htm

```diff
@@ -26,6 +26,11 @@
 			<tbody>
 				<tr>
 					<td id="beta" >
+						<iframe id="Beta" src="Beta/Beta.htm" title="FLEx Bridge">
+							<p>Your browser does not support iframes.</p>
+						</iframe>
+					</td>
+					<td id="alpha" >
```

Comment:
I was thinking that they would be listed from more to less stable in left to right order. So, I'm open to suggestions, but I did do it on purpose.